coderd/users.go

```diff
@@ -1165,8 +1165,8 @@ func convertUsers(users []database.User, organizationIDsByUserID map[uuid.UUID][
 
 func userOrganizationIDs(ctx context.Context, api *API, user database.User) ([]uuid.UUID, error) {
 	organizationIDsByMemberIDsRows, err := api.Database.GetOrganizationIDsByMemberIDs(ctx, []uuid.UUID{user.ID})
-	if errors.Is(err, sql.ErrNoRows) || len(organizationIDsByMemberIDsRows) == 0 {
```

Comment:
I understand that `len(organizationIDsByMemberIDsRows) == 0` will never happen against the real database, and `dbfake.go` implementation handles this case:

```go
	if len(getOrganizationIDsByMemberIDRows) == 0 {
		return nil, sql.ErrNoRows
	}
```

Reply:
Found a panic from this when missing org ids. The problem is that real db queries only return `sql.ErrNoRows` on `:one` queries, while `:many` just return zero rows without error.